backends/p4fmt/p4fmt.cpp

```diff
@@ -24,9 +25,8 @@ std::stringstream getFormattedOutput(std::filesystem::path inputFile) {
         return formattedOutput;
     }
 
-    auto top4 = P4::ToP4(&formattedOutput, false);
-    // Print the program before running front end passes.
-    program->apply(top4);
+    PassManager passes({new P4::P4Fmt::Attach(), new P4::ToP4(&formattedOutput, false)});
```

Comment:
You do not really need a pass manager here, you can just do 
`program = program->apply(P4::P4Fmt::Attach())` then print it or write it to a file.